troposphere/__init__.py

```diff
@@ -160,12 +160,14 @@ def JSONrepr(self):
                 raise ValueError(
                     "Resource %s required in type %s" % (k, rtype))
         self.validate()
-        # If no other properties are set, only return the Type.
         # Mainly used to not have an empty "Properties".
         if self.properties:
             return self.resource
         elif hasattr(self, 'resource_type'):
-            return {'Type': self.resource_type}
+            return {
+                k: v for k, v in self.resource.items()
```

Comment:
Hi @simonschmidt,

This change has caused Troposphere to throw errors when using python 2.6 as it doesn't support dictionary comprehensions.

Would it possible to fix this per chance?

Thanks!